src/io/flutter/logging/FlutterLogPreferences.java

```diff
@@ -18,7 +18,7 @@
   public boolean SHOW_LOG_CATEGORY = true;
   public boolean SHOW_COLOR = true;
 
-  public int TOOL_WINDOW_LOG_LEVEL = 800; // Level INFO
+  public int TOOL_WINDOW_LOG_LEVEL = FlutterLog.Level.NONE.value;
```

Comment:
Looking at this again I realize that the naming confuses me.  In general, I think ALL_CAPS are reserved for static finals.

Maybe consider changing these?

Reply:
👍 
I updated.